### DISCUSSION TITLE
some error

### DISCUSSION THREAD

**Discussion by xclidongbo** [fastlane/fastlane#17105] [Q&A]
[10:14:12]: Connection reset by peer - SSL_connect

+------+-----------------------------------+-------------+
|                    fastlane summary                    |
+------+-----------------------------------+-------------+
| Step | Action                            | Time (in s) |
+------+-----------------------------------+-------------+
| 1    | default_platform                  | 0           |
| 2    | Switch to ios ios_etp lane        | 0           |
| 3    | Switch to ios RNHandle lane       | 0           |
| 4    | Switch to ios buildExportIpa lane | 0           |
| 5    | increment_build_number            | 2           |
| 6    | get_info_plist_value              | 0           |
| 7    | get_info_plist_value              | 0           |
| 8    | get_info_plist_value              | 0           |
| 9    | build_app                         | 950         |
| 10   | last_git_commit                   | 0           |
| 💥   | fir_cli                           | 13          |
+------+-----------------------------------+-------------+

[10:14:12]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

➡️  Connection reset by peer - get_provisioning_profile
    https://github.com/fastlane/fastlane/issues/15051 [closed] 9 💬
    10 Dec 2019

➡️  Connection reset by peer - SSL_connect
    https://github.com/fastlane/fastlane/issues/14240 [closed] 1 💬
    17 Apr 2019

➡️  Connection reset by peer - SSL_connect
    https://github.com/fastlane/fastlane/issues/13460 [closed] 4 💬
    15 Jan 2019

and 14 more at: https://github.com/fastlane/fastlane/search?q=Connection%20reset%20by%20peer%20-%20SSL_connect&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
Traceback (most recent call last):
	40: from /usr/local/lib/ruby/gems/2.6.0/bin/fastlane:23:in `<main>'
	39: from /usr/local/lib/ruby/gems/2.6.0/bin/fastlane:23:in `load'
	38: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/bin/fastlane:23:in `<top (required)>'
	37: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/cli_tools_distributor.rb:119:in `take_off'
	36: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/commands_generator.rb:41:in `start'
	35: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/commands_generator.rb:352:in `run'
	34: from /usr/local/lib/ruby/gems/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	33: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	32: from /usr/local/lib/ruby/gems/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	31: from /usr/local/lib/ruby/gems/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	30: from /usr/local/lib/ruby/gems/2.7.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	29: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/commands_generator.rb:108:in `block (2 levels) in run'
	28: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle'
	27: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane'
	26: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:45:in `execute'
	25: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	24: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	23: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/lane.rb:33:in `call'
	22: from Fastfile:221:in `block (2 levels) in parsing_binding'
	21: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing'
	20: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name'
	19: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:229:in `execute_action'
	18: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:229:in `chdir'
	17: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action'
	16: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	15: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-2.157.0/fastlane/lib/fastlane/runner.rb:263:in `block (2 levels) in execute_action'
	14: from /usr/local/lib/ruby/gems/2.7.0/gems/fastlane-plugin-fir_cli-2.0.9/lib/fastlane/plugin/fir_cli/actions/fir_cli_action.rb:39:in `run'
	13: from /usr/local/lib/ruby/gems/2.7.0/gems/fir-cli-2.0.9/lib/fir/util/publish.rb:31:in `publish'
	12: from /usr/local/lib/ruby/gems/2.7.0/gems/fir-cli-2.0.9/lib/fir/util/publish.rb:247:in `wxwork_notifier'
	11: from /usr/local/lib/ruby/gems/2.7.0/gems/api_tools-0.1.0/lib/api_tools/default_rest_module.rb:19:in `block (2 levels) in <module:DefaultRestModule>'
	10: from /usr/local/lib/ruby/gems/2.7.0/gems/api_tools-0.1.0/lib/api_tools/default_rest_module.rb:64:in `basic_request'
	 9: from /usr/local/lib/ruby/gems/2.7.0/gems/api_tools-0.1.0/lib/api_tools/default_rest_module.rb:64:in `times'
	 8: from /usr/local/lib/ruby/gems/2.7.0/gems/api_tools-0.1.0/lib/api_tools/default_rest_module.rb:66:in `block in basic_request'
	 7: from /usr/local/lib/ruby/gems/2.7.0/gems/rest-client-2.1.0/lib/restclient/request.rb:63:in `execute'
	 6: from /usr/local/lib/ruby/gems/2.7.0/gems/rest-client-2.1.0/lib/restclient/request.rb:163:in `execute'
	 5: from /usr/local/lib/ruby/gems/2.7.0/gems/rest-client-2.1.0/lib/restclient/request.rb:727:in `transmit'
	 4: from /usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/net/http.rb:932:in `start'
	 3: from /usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/net/http.rb:943:in `do_start'
	 2: from /usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/net/http.rb:1009:in `connect'
	 1: from /usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/net/protocol.rb:44:in `ssl_socket_connect'
/usr/local/Cellar/ruby/2.7.1_2/lib/ruby/2.7.0/net/protocol.rb:44:in `connect_nonblock': \e[31m[!] Connection reset by peer - SSL_connect\e[0m (Errno::ECONNRESET)

**Comment by Thompsons21** [comment]
After hours of trying different things, I finally ran:

`sudo gem clean`
And now it works as expected -_-

Putting this here in case it saves anyone else the frustration! [KohlsFeedback Survey](https://www.kohlsfeedback.onl/)
